README.md

```diff
@@ -0,0 +1,82 @@
+# RDS Scheduler
+
+[![Build_Status](https://circleci.com/gh/appvia/rds-scheduler.svg?style=svg)](https://circleci.com/gh/appvia/rds-scheduler) [![Docker Repository on Quay](https://quay.io/repository/appvia/rds-scheduler/status "Docker Repository on Quay")](https://quay.io/repository/appvia/rds-scheduler)
+
+Manage uptime schedules for RDS Instances and shutdown instances outside of working hours.
+
+All RDS instances are checked for a given AWS Tag, `appvia.io/rds-scheduler/uptime-schedule`, to determine whether they need to be managed according to a specified uptime schedule. If the AWS Tag is not found, no action is taken on that DB instance.
+
+The value of an AWS Tag should hold a time definition matching the pattern: `<WEEKDAY-FROM>-<WEEKDAY-TO> <HH:MM-FROM>-<HH:MM-TO> <TIMEZONE>`
+
+For example:
+```yml
+appvia.io/rds-scheduler/uptime-schedule: MON-FRI 08:30-18:00 Europe/London
+```
+
+The above definition would start an RDS instance at 08:30 and shut it down at 18:00 on weekdays only, leaving the instance in a stopped state in the evenings and weekends.
+
+## Usage
+
+Set the Tag `appvia.io/rds-scheduler/uptime-schedule` on each RDS instance, providing a time definition to keep the RDS instance online for.
+
+Run the docker container, providing AWS Credentials either as environment variables or mounting in your AWS config directory, e.g.:
+
+```bash
+# Pass as environment variables
+docker run --rm -t -e AWS_ACCESS_KEY_ID=X AWS_SECRET_ACCESS_KEY=X -e AWS_REGION=eu-west-2 quay.io/appvia/rds-scheduler
+
+# Use AWS config and profile
+docker run --rm -t -v ~/.aws:/home/app/.aws:ro -e AWS_PROFILE=my-aws-profile quay.io/appvia/rds-scheduler
+```
+
+### Configuration
+
+The following environment variables can be passed:
+- `DRY_RUN`: Don't make any changes to RDS instances, just prints what actions would be performed to stdout (default: `false`)
+- `LOOP_INTERVAL`: How frequently (in seconds) to loop and perform checks on the RDS instance schedules (default: `30`)
```

Comment:
It may be worth elaborating this to something like `LOOP_INTERVAL_SECS`, just to make it clearer (without having to look up docs).